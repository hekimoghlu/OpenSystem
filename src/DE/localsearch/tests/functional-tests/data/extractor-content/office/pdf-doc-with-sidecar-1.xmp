<?xpacket begin="ï»¿" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="XMP Core 4.4.0-Exiv2">
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
  <rdf:Description rdf:about=""
    xmlns:xmpMM="http://ns.adobe.com/xap/1.0/mm/"
    xmlns:stEvt="http://ns.adobe.com/xap/1.0/sType/ResourceEvent#"
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:GIMP="http://www.gimp.org/xmp/"
    xmlns:pdf="http://ns.adobe.com/pdf/1.3/"
    xmlns:cc="http://creativecommons.org/ns#"
    xmlns:Iptc4xmpCore="http://iptc.org/std/Iptc4xmpCore/1.0/xmlns/"
    xmlns:iptcExt="http://iptc.org/std/Iptc4xmpExt/2008-02-29/"
    xmlns:xmp="http://ns.adobe.com/xap/1.0/"
    xmlns:xmpRights="http://ns.adobe.com/xap/1.0/rights/"
    xmlns:tiff="http://ns.adobe.com/tiff/1.0/"
    xmlns:exif="http://ns.adobe.com/exif/1.0/"
    xmlns:photoshop="http://ns.adobe.com/photoshop/1.0/"
   xmpMM:DocumentID="gimp:docid:gimp:d78b2f95-ab2d-4f8c-b741-1f47c7da3222"
   xmpMM:InstanceID="xmp.iid:a665c4af-ccd0-4a4a-81e1-240b2172ce51"
   xmpMM:OriginalDocumentID="xmp.did:328b61c3-e8af-49be-83d2-d34b1c3aa629"
   dc:Format="image/png"
   GIMP:API="3.0"
   GIMP:Platform="Linux"
   GIMP:TimeStamp="1714323335985819"
   GIMP:Version="2.99.19"
   Iptc4xmpCore:CiAdrCity="Amsterdam"
   Iptc4xmpCore:CiAdrCtry="Netherlands"
   Iptc4xmpCore:CiAdrExtadr="Address"
   Iptc4xmpCore:CiAdrPcode="0001aa"
   Iptc4xmpCore:CiAdrRegion="North Holland"
   Iptc4xmpCore:CiEmailWork="a@example.com"
   Iptc4xmpCore:CiTelWork="+3100000000"
   Iptc4xmpCore:CiUrlWork="https://example.com/"
   Iptc4xmpCore:IntellectualGenre="Nihilism"
   Iptc4xmpCore:Location="Sublocation yadda yadda"
   iptcExt:City="city yadda yadda"
   iptcExt:CountryCode="NL"
   iptcExt:CountryName="country yadda yadda"
   iptcExt:ProvinceState="state yadda yadda"
   iptcExt:Sublocation="Sublocation yadda yadda"
   iptcExt:WorldRegion="Europe"
   xmp:CreatorTool="GIMP"
   xmp:MetadataDate="2024:04:28T18:55:35+02:00"
   xmp:Rating="5"
   xmp:ModifyDate="2024:04:28T18:55:35+02:00"
   xmp:CreateDate="2015-12-29T11:26:53.199"
   xmpRights:Marked="False"
   xmpRights:WebStatement="http://example.com"
   tiff:ImageWidth="10"
   tiff:ImageLength="10"
   tiff:Orientation="1"
   tiff:YCbCrPositioning="1"
   tiff:XResolution="72/1"
   tiff:YResolution="72/1"
   tiff:ResolutionUnit="2"
   tiff:Make="Apple"
   tiff:Model="iPhone 5"
   tiff:Software="GIMP 2.99.19"
   exif:ExifVersion="0221"
   exif:FlashpixVersion="0100"
   exif:ColorSpace="1"
   exif:PixelXDimension="10"
   exif:PixelYDimension="10"
   exif:ExposureTime="1/10870"
   exif:FNumber="12/5"
   exif:ExposureProgram="2"
   exif:ShutterSpeedValue="15741/1174"
   exif:ApertureValue="4845/1918"
   exif:BrightnessValue="7792/651"
   exif:ExposureBiasValue="0/1"
   exif:MeteringMode="5"
   exif:FocalLength="103/25"
   exif:SensingMethod="2"
   exif:SceneType="1"
   exif:CustomRendered="2"
   exif:ExposureMode="0"
   exif:WhiteBalance="0"
   exif:FocalLengthIn35mmFilm="33"
   exif:SceneCaptureType="0"
   exif:GPSLatitude="0,0.0000000N"
   exif:GPSLongitude="0,0.0000000W"
   exif:GPSAltitudeRef="0"
   exif:GPSAltitude="6564/100"
   exif:GPSTimeStamp="2015-12-29T11:26:51.360000000"
   exif:GPSSpeedRef="K"
   exif:GPSSpeed="0/1"
   exif:GPSImgDirectionRef="T"
   exif:GPSImgDirection="31855/186"
   exif:GPSDestBearingRef="T"
   exif:GPSDestBearing="31855/186"
   exif:Artist="Artist"
   photoshop:DateCreated="2015-12-29T11:26:53+00:00"
   photoshop:Urgency="1"
   photoshop:Category="Category"
   photoshop:Instructions="Instructions"
   photoshop:AuthorsPosition="Author title"
   photoshop:TransmissionReference="Identifier"
   photoshop:Headline="Headline!"
   photoshop:Credit="Credit line"
   photoshop:Source="Source"
   photoshop:CaptionWriter="That's me!">
   <xmpMM:History>
    <rdf:Seq>
     <rdf:li
      stEvt:action="saved"
      stEvt:changed="/metadata"
      stEvt:instanceID="xmp.iid:60ac053a-d78f-4dd2-98af-5d10d1424988"
      stEvt:softwareAgent="GIMP 2.99.19 (Linux)"
      stEvt:when="2024-04-28T18:36:41+02:00"/>
     <rdf:li
      stEvt:action="saved"
      stEvt:changed="/"
      stEvt:instanceID="xmp.iid:f9dabd40-cfa4-490f-9a4f-770cbbb788bb"
      stEvt:softwareAgent="GIMP 2.99.19 (Linux)"
      stEvt:when="2024-04-28T18:55:35+02:00"/>
    </rdf:Seq>
   </xmpMM:History>
   <dc:title>
    <rdf:Alt>
     <rdf:li xml:lang="x-default">Photograph</rdf:li>
    </rdf:Alt>
   </dc:title>
   <dc:subject>
    <rdf:Bag>
     <rdf:li>subject</rdf:li>
    </rdf:Bag>
   </dc:subject>
   <dc:keywords>
    <rdf:Bag>
     <rdf:li>amazing, bright</rdf:li>
    </rdf:Bag>
   </dc:keywords>
   <pdf:keywords>
    <rdf:Bag>
     <rdf:li>amazing2, bright2</rdf:li>
    </rdf:Bag>
   </pdf:keywords>
   <dc:creator>
    <rdf:Seq>
     <rdf:li>Author</rdf:li>
    </rdf:Seq>
   </dc:creator>
   <dc:rights>
    <rdf:Alt>
     <rdf:li xml:lang="x-default">Copyright</rdf:li>
    </rdf:Alt>
   </dc:rights>
   <dc:description>
    <rdf:Alt>
     <rdf:li xml:lang="x-default">Photograph description</rdf:li>
    </rdf:Alt>
   </dc:description>
   <dc:publisher>
     <rdf:Seq>
       <rdf:li>Publisher</rdf:li>
     </rdf:Seq>
   </dc:publisher>
   <dc:type>
     <rdf:Seq>
       <rdf:li>Type</rdf:li>
     </rdf:Seq>
   </dc:type>
   <dc:identifier>
     <rdf:Seq>
       <rdf:li>Identifier</rdf:li>
     </rdf:Seq>
   </dc:identifier>
   <dc:source>
     <rdf:Seq>
       <rdf:li>http://example.com/</rdf:li>
     </rdf:Seq>
   </dc:source>
   <dc:language>
     <rdf:Seq>
       <rdf:li>en_US</rdf:li>
     </rdf:Seq>
   </dc:language>
   <dc:relation>
     <rdf:Seq>
       <rdf:li>Relation</rdf:li>
     </rdf:Seq>
   </dc:relation>
   <dc:coverage>
     <rdf:Seq>
       <rdf:li>Coverage</rdf:li>
     </rdf:Seq>
   </dc:coverage>
   <cc:license>
     <rdf:Seq>
       <rdf:li>License</rdf:li>
     </rdf:Seq>
   </cc:license>
   <Iptc4xmpCore:CreatorContactInfo
    Iptc4xmpCore:CiAdrCity="Amsterdam"
    Iptc4xmpCore:CiAdrCtry="Netherlands"
    Iptc4xmpCore:CiAdrExtadr="Address"
    Iptc4xmpCore:CiAdrPcode="0001aa"
    Iptc4xmpCore:CiAdrRegion="North Holland"
    Iptc4xmpCore:CiEmailWork="a@example.com"
    Iptc4xmpCore:CiTelWork="+3100000000"
    Iptc4xmpCore:CiUrlWork="https://example.com/"/>
   <Iptc4xmpCore:Scene>
    <rdf:Bag>
     <rdf:li>what is this</rdf:li>
    </rdf:Bag>
   </Iptc4xmpCore:Scene>
   <Iptc4xmpCore:SubjectCode>
    <rdf:Bag>
     <rdf:li>What is this again</rdf:li>
    </rdf:Bag>
   </Iptc4xmpCore:SubjectCode>
   <iptcExt:PersonInImage>
    <rdf:Bag>
     <rdf:li>All that is good in the world</rdf:li>
    </rdf:Bag>
   </iptcExt:PersonInImage>
   <xmpRights:UsageTerms>
    <rdf:Alt>
     <rdf:li xml:lang="x-default">Usage terms go here</rdf:li>
    </rdf:Alt>
   </xmpRights:UsageTerms>
   <tiff:BitsPerSample>
    <rdf:Seq>
     <rdf:li>8 8 8</rdf:li>
    </rdf:Seq>
   </tiff:BitsPerSample>
   <exif:ComponentsConfiguration>
    <rdf:Seq>
     <rdf:li>1</rdf:li>
     <rdf:li>2</rdf:li>
     <rdf:li>3</rdf:li>
     <rdf:li>0</rdf:li>
    </rdf:Seq>
   </exif:ComponentsConfiguration>
   <exif:ISOSpeedRatings>
    <rdf:Seq>
     <rdf:li>50</rdf:li>
    </rdf:Seq>
   </exif:ISOSpeedRatings>
   <exif:Flash
    exif:Fired="False"
    exif:Return="0"
    exif:Mode="2"
    exif:Function="False"
    exif:RedEyeMode="False"/>
   <exif:SubjectArea>
    <rdf:Seq>
     <rdf:li>1631 1223 1795 1077</rdf:li>
    </rdf:Seq>
   </exif:SubjectArea>
   <photoshop:SupplementalCategories>
    <rdf:Bag>
     <rdf:li>Supplemental category</rdf:li>
    </rdf:Bag>
   </photoshop:SupplementalCategories>
  </rdf:Description>
 </rdf:RDF>
</x:xmpmeta>
<?xpacket end="w"?>