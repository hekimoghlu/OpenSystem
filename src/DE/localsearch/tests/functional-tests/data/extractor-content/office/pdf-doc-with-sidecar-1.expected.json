{
    "test": {
        "Filename": "pdf-doc-with-sidecar-1.pdf",
        "Comment": "PDF document with XMP sidecar"
    },
    "metadata": {
        "@graph": [
	    {
                "dc:format" : "image/png",
                "@type" : "nfo:PaginatedTextDocument",
                "slo:location" : {
                    "slo:latitude" : "0",
                    "slo:longitude" : "-0",
                    "@type" : "slo:GeoLocation",
                    "slo:altitude" : "65.640000000000001"
                },
                "nco:creator" : {
                    "@id" : "urn:contact:first%20last",
                    "nco:fullname" : "first last",
                    "@type" : "nco:Contact"
                },
                "nco:publisher": {
                  "@id": "urn:contact:Publisher",
                  "nco:fullname": "Publisher",
                  "@type": "nco:Contact"
                },
                "dc:type": "Type",
                "dc:identifier": "Identifier",
                "dc:source": "http://example.com/",
                "dc:language": "en_US",
                "dc:relation": "Relation",
                "dc:coverage": "Coverage",
                "nie:license": "License",
                "nie:subject": "subject",
                "nie:description" : "Photograph description",
                "nfo:pageCount" : 1,
                "nie:title" : "Photograph",
                "nie:copyright" : "Copyright",
                "nie:contentCreated" : "2024-04-29T00:10:20+02",
                "nie:plainTextContent" : "Hello world ",
                "nao:hasTag": [
                    {
                        "@id": "urn:tag:amazing",
                        "nao:prefLabel": "amazing",
                        "@type": "nao:Tag"
                    },
                    {
                        "@id": "urn:tag:bright",
                        "nao:prefLabel": "bright",
                        "@type": "nao:Tag"
                    },
                    {
                        "@id": "urn:tag:amazing2",
                        "nao:prefLabel": "amazing2",
                        "@type": "nao:Tag"
                    },
                    {
                        "@id": "urn:tag:bright2",
                        "nao:prefLabel": "bright2",
                        "@type": "nao:Tag"
                    }
                ],
                "nco:contributor": {
                    "@id": "urn:contact:Artist",
                    "nco:fullname": "Artist",
                    "@type": "nco:Contact"
                }
            }
        ]
    }
}
